src/rprblender/nodes/__init__.py

```diff
@@ -64,31 +68,46 @@ def poll(cls, context):
         NodeItem('RPRShaderNodePassthrough'),
         NodeItem('RPRShaderNodeLayered'),
         NodeItem('RPRShaderNodeToon'),
-    ]),
-    RPR_ShaderNodeCategory("RPR_TEXTURES", "Texture", items=[
+        NodeItem('ShaderNodeBsdfAnisotropic'),
+        NodeItem('ShaderNodeBsdfDiffuse'),
+        NodeItem('ShaderNodeBsdfGlass'),
+        NodeItem('ShaderNodeBsdfGlossy'),
+        NodeItem('ShaderNodeBsdfRefraction'),
+        NodeItem('ShaderNodeBsdfTranslucent'),
+        NodeItem('ShaderNodeBsdfTransparent'),
+        NodeItem('ShaderNodeBsdfVelvet'),
+        NodeItem('ShaderNodeSubsurfaceScattering'),
+        NodeItem('RPRShaderNodeDiffuse'),
+        NodeItem('RPRShaderNodeEmissive'),
+        NodeItem('RPRShaderNodeBlend'),
+    ])),
```

Comment:
So now I'm remembering why these are not included.  We wanted to suggest that people use Principled and Uber with the RPR plugin.  If they have node networks already made they still work.

Reply:
These will be removed:
RPRShaderNodeDiffuse
RPRShaderNodeEmissive
RPRShaderNodeBlend

How about these nodes?
RPRShaderNodeImageTexture
RPRShaderNodeBumpMap
RPRShaderNodeNormalMap

Reply:
The reason the RPR ones are not there is because they're redundant to the blender ones

Reply:
Removed.
RPRShaderNodeImageTexture
RPRShaderNodeBumpMap
RPRShaderNodeNormalMap